content/shmem_atomic_fetch_xor_nbi.tex

```diff
@@ -0,0 +1,58 @@
+\color{Green}
+\apisummary{
+  This nonblocking atomic operation performs an atomic fetching bitwise XOR
+  operation on a remote data object.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+void @\FuncDecl{shmem\_atomic\_fetch\_xor\_nbi}@(TYPE *fetch, TYPE *dest, TYPE value, int pe);
+void @\FuncDecl{shmem\_atomic\_fetch\_xor\_nbi}@(shmem_ctx_t ctx, TYPE *fetch, TYPE *dest, TYPE value, int pe);
+\end{C11synopsis}
+where \TYPE{} is one of the bitwise \ac{AMO} types specified by
+Table~\ref{bitamotypes}.
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_\FuncParam{TYPENAME}\_atomic\_fetch\_xor\_nbi}@(TYPE *fetch, TYPE *dest, TYPE value, int pe);
+void @\FuncDecl{shmem\_ctx\_\FuncParam{TYPENAME}\_atomic\_fetch\_xor\_nbi}@(shmem_ctx_t ctx, TYPE *fetch, TYPE *dest, TYPE value, int pe);
+\end{Csynopsis}
+where \TYPE{} is one of the bitwise \ac{AMO} types and has a corresponding
+\TYPENAME{} specified by Table~\ref{bitamotypes}.
+
+\begin{apiarguments}
+
+  \apiargument{IN}{ctx}{A context handle specifying the context on which to
+      perform the operation. When this argument is not provided, the
+      operation is performed on the default context.}
+  \apiargument{OUT}{fetch}{Local data object to be updated.}
+  \apiargument{OUT}{dest}{A pointer to the remotely accessible data object to
+    be updated.}
+  \apiargument{IN}{value}{The operand to the bitwise XOR operation.}
+  \apiargument{IN}{pe}{An integer value for the \ac{PE} on which \VAR{dest}
+    is to be updated.}
+
+\end{apiarguments}
+
+\apidescription{
+    The nonblocking \FUNC{shmem\_atomic\_fetch\_xor\_nbi} routines performs an
```

Comment:
Replace "performs" with "perform" here and below in the same paragraph.

Reply:
fixed